docs/community/contributing.md

```diff
@@ -33,7 +33,7 @@ We follow the [SymVer](https://semver.org/) versioning strategy: \[MAJOR\].\[MIN
 * For non-breaking feature changes, \[MINOR\] should be incremented.  This can be accomplished by running `./version.sh -u -n`
 * For major and/or breaking changes, \[MAJOR\] should be incremented.  This can be accomplished by running `./version.sh -u -m`
 
-To ensure that all product versioning is consistent, our CI builds will execute `./version.sh -c` to check all known instances of version in our YAML, TOML, and code. This will also check to make sure that version.txt has been changed. If a pull request is needed where the version should not be changed, include `[SAME VERSION]` in the pull request title.
+To ensure that all product versioning is consistent, our CI builds will execute `./version.sh -c` to check all known instances of version in our YAML, TOML, and code. This will also check to make sure that version.txt has been changed. If a pull request is needed where the version should not be changed, add `same version` label to the pull request.
```

Comment:
maybe add "by commenting `/add-same-version-label`". I know it is mentioned later but would help to reiterate

Reply:
Sounds good!